modules/ROOT/pages/update-acm.adoc

```diff
@@ -8,6 +8,15 @@ Keep your Anypoint API Community Manager instance up to date to take advantage o
 
 For information about updates, see xref:notifications.adoc[Important Notifications].
 
+After you install an update with this process, it is not possible to undo the installation and return to a previous version. We recommend installing updates with this best practice workflow:
+
+. Create a developer sandbox.
+. Install the update in the developer sandbox and test it.
+. Discard the developer sandbox.
+. Keep a partial copy sandbox or a full copy sandbox as a staging environment.
```

Comment:
Should this step precede the one before it? What is the partial or full copy a copy of? Is that clear to the audience?

Reply:
Maybe it would be more clear to say Create a staging sandbox?

Reply:
Yes, I think it's better to move this staging sandbox step before the developer sandbox step. It will have a partial or full copy of the production data, and that should be clear to the audience. Fixed. Thank you!